pom.xml

```diff
@@ -56,6 +56,12 @@
       <version>5.5</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>1.7.30</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
This change already exists in main (but using logback instead, so we can occasionally turn on DEBUG level, which slf4j-simple doesn't support).